src/main/java/org/opentripplanner/netex/mapping/QuayMapper.java

```diff
@@ -1,5 +1,6 @@
 package org.opentripplanner.netex.mapping;
 
+import jakarta.validation.constraints.NotNull;
```

Comment:
The rest of the codebase uses @javax.annotation.Nonnull rather than @jakarta.validation.constraints.NotNull

Reply:
Oh good point, I will change that